internal/compress/zstd/zstd.go

```diff
@@ -21,8 +21,13 @@ import (
 	"github.com/klauspost/compress/zstd"
 )
 
-// EOption is type alias of zstd.EOption.
-type EOption = zstd.EOption
+type (
+	// EOption is type alias of zstd.EOption.
+	EOption = zstd.EOption
+
+	// DOption is type alias of zstd.DOption.
+	DOption = zstd.DOption
```

Comment:
Why did you create 2 same type aliases?

Reply:
It is because the external zstd package contains 2 different options for different target, one is for `zstd.Encoder` and the another one is for `zstd.Decocder. 
We need to create both of them for initializing.